addons/orchestrator-cluster/templates/cluster.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: Cluster
+metadata:
+  name: {{ include "kblib.clusterName" . }}
+  labels: {{ include "kblib.clusterLabels" . | nindent 4 }}
+  annotations:
+    kubeblocks.io/enabled-node-port-svc: orchestrator,mysql,proxysql
+    kubeblocks.io/enabled-pod-ordinal-svc: orchestrator,mysql,proxysql
```

Comment:
mysql and proxsql are not even specified in this cluster, is it necessary to specify them here?